### ISSUE TITLE
[Feature Request] let qbitttorrent-api fetch torrents from urls

### ISSUE THREAD

**Issue by bodysoda69** [yash-dk/TorToolkit-Telegram#87]
It'd be nice to have qbitttorrent fetch the torrent inside the urls and upload the downloaded file to telegram instead of uploading the torrent file. 

Hope this helps - https://qbittorrent-api.readthedocs.io/en/latest/apidoc/torrents.html

**Comment by yash-dk** [comment]
This can't be done easily as ttk automatically identifies if it's a direct link or torrent link.. giving all to qbit will not work. Will work in the future.

**Comment by bodysoda69** [comment]
Yeah, it's hard unless ttk has a separate commands for torrent and ddl like instadl, ytdl. Providing separate commands will always be useful in the future for further updates too.

**Comment by gitenvx** [comment]
Add manual the site is done

**Comment by bodysoda69** [comment]
> Add manual the site is done

Speak in english please.
